crates/ef-testing/src/models/case.rs

```diff
@@ -123,6 +123,24 @@ impl BlockchainTestCase {
             .and_then(|transaction| transaction.gas_price)
             .map(|gas_price| gas_price.0)
             .unwrap_or_default();
+        let max_priority_fee_per_gas = maybe_transaction
+            .and_then(|transaction| transaction.max_priority_fee_per_gas)
+            .map(|max_priority_fee_per_gas| max_priority_fee_per_gas.0)
+            .unwrap_or_default();
+        let max_fee_per_gas = maybe_transaction
```

Comment:
this is not `max_fee_per_gas` anymore, it's confusing to keep the naming. We should keep `max_fee_per_gas` as the tx, and and name the result `effective_gas_price` as in the EIP spec `(= base_fee_per_gas + priority_fee_per_gas)`